update-assets-docker.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
It's probably worth just changing `update-assets.sh` to use this script? and then we can all try running it as an additional test on our machines, and check it's deterministic?

Reply:
actually....I can update `update-assets.sh` so that it supports both modes. local and docker. will go that way

Reply:
done. merged both scripts into one. please do let me know if you wanna go that way or if you just prefer the docker approach